packages/react/src/globals/_mixins.scss

```diff
@@ -41,3 +42,21 @@
     border-bottom-#{$last-button-side}-radius: $border-radius;
   }
 }
+
+@mixin list-box-menu-fits-content() {
+  .#{$prefix}--list-box__menu {
```

Comment:
Is this `prefix` b/c you're overriding carbon styles or should it be `iotPrefix`?

Reply:
I needed to override the Carbon width